packages/compass-sidebar/package.json

```diff
@@ -1,149 +1,119 @@
 {
   "name": "@mongodb-js/compass-sidebar",
-  "productName": "Sidebar plugin",
```

Comment:
Ah, damn, I think we forgot to add this to create-workspace script. Can you add it back (and optionally update the script to include it, otherwise I'll do it a bit later)

Reply:
Added it back in and added the following to the script:

```
    productName: `${name} Plugin`,
```

That's not going to produce perfect package names, but at least then we won't forget about it. I guess we can try and change dashes to spaces and do Title Case, but not sure how much that's worth. I mean how often do we run this script? It also still wouldn't help for changing `compass-sidebar` into `Sidebar` anyway.

Reply:
Yeah, this quick fix makes a ton of sense, this is mostly a leftover for the screen that we are planning to remove from compass anyway, then we can clean all of them up. If the name there will make some sense, that's already perfect 😄